examples/inspiral/run.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Test pycbc inspiral by running over GW150914 with a limited template bank
+albert=albert.phys.uwm.edu/download
```

Comment:
That seems sane, I was just keeping it where it had been, but it would be better to have this in a a repo, or pull the file directly from LOSC in the case of the frame file.

Reply:
This doesn't seem resolved yet?